languages/javascript/exercises/concept/errors/.docs/introduction.md

```diff
@@ -0,0 +1,29 @@
+Errors are useful to report when something is wrong or unexpected in a program or a piece of code.
+
+They are javascript objects.
+
+The main property of this object is `message`:
+
+```javascript
+const error = new Error('Oops, something went wrong')
+
+error.message
+// => "Oops, something went wrong"
+```
+
+Errors are special objects that :
```

Comment:
We probably want to elaborate on why they're special. (They capture stacktrace, etc.). 

However it currently reads as if only error can be thrown. This is not true. _Anything_ can be thrown via `throw` and caught via `catch` in JavaScript.

Reply:
Yes i know that, but i thought that the shortcut was not harmful 🤔

Reply:
Got it! Sounds good.

Reply:
5a446cf52034d79933364d6431bf479cfe4c86b1